science/osmocore/Portfile

```diff
@@ -71,6 +70,7 @@ depends_build-append \
     port:pkgconfig
 
 depends_lib-append \
+    path:lib/libusb-1.0.dylib \
```

Comment:
Needs to be more like `path:lib/libusb-1.0.dylib:libusb`.

Reply:
d638bb675c8 (I somehow managed to not completely copy the example out of another port…)